libkbfs/bserver_remote.go

```diff
@@ -332,23 +342,26 @@ func (b *BlockServerRemote) Get(ctx context.Context, tlfID tlf.ID, id kbfsblock.
 
 // Put implements the BlockServer interface for BlockServerRemote.
 func (b *BlockServerRemote) Put(ctx context.Context, tlfID tlf.ID, id kbfsblock.ID,
-	context kbfsblock.Context, buf []byte,
+	bContext kbfsblock.Context, buf []byte,
 	serverHalf kbfscrypto.BlockCryptKeyServerHalf) (err error) {
+	if b.config.DiskBlockCache() != nil {
+		go b.config.DiskBlockCache().Put(context.TODO(), tlfID, id, buf, serverHalf)
```

Comment:
why not reuse existing context? if the put is cancelled, we'd still want to cancel the cache put, right?